libsatadrivers/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2019, DornerWorks
+# Copyright 2018, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+cmake_minimum_required(VERSION 3.7.2)
+
+project(libsatadrivers C)
+
+set(configure_string "")
+
+config_option(LibSatadrivers LIB_SATA "Support library for x86 SATA" DEFAULT OFF)
```

Comment:
What is this config used for? Is it still required?

Reply:
Nope, that's not used. We can remove it.